features/contents-attr.yml

```
@@ -0,0 +1,6 @@
+name: "content: attr()"
+description: "The `content: attr()` CSS declaration sets a `::before` or `::after` pseudo-element's content to the value of the specified HTML attribute."
```

For consistency with the other description:

```suggestion
description: "The `attr()` CSS function, in `content` property declarations, sets a `::before` or `::after` pseudo-element's content to the value of the specified HTML attribute."
```